test/integration/targets/eos_eapi/tests/cli/off.yaml

```diff
@@ -10,6 +10,7 @@
   register: eos_eapi_output
   connection: local
 
+- pause: seconds=20
```

Comment:
when the service is disabled via command line, the system will gracefully shut it down... with the new connection plugin we iterate to quickly to the next task before the system has had a chance to disable the service so we need to briefly pause.   In retro spec, we should probably check for this condition in the module code and not just assume the service fully transitioned to stopped.